rpc/context.go

```diff
@@ -0,0 +1,100 @@
+package rpc
```

Comment:
could we add some notes at the top of the file or a different name of the file to know that this is a Quorum specific file?

Reply:
added